_posts/2019/2019-08-20-end-to-end-tests-with-geb-and-spock.md

```diff
@@ -0,0 +1,164 @@
+---
+title: End-to-end tests with Geb and Spock
+layout: post
+date: '2019-08-20'
+header_image: public/crash-test.jpg
+header_position: middle
```

Comment:
No, it is supposed to show the middle part of the picture. I cropped it myself, and I spent some time doing that, because you also want it in a certain size for no apparent reason.

Reply:
We don't support `middle` in the front matter, but `center` does the trick. It's still fine :)